hibernate-reactive-core/src/main/java/org/hibernate/reactive/adaptor/impl/ResultSetAdaptor.java

```diff
@@ -375,6 +386,7 @@ public int getColumnCount() {
 
 			@Override
 			public int getColumnType(int column) {
+//				if( rows.columnDescriptors().isEmpty() ) return 0;
```

Comment:
What's this comment?

Reply:
Just tried something but commented it out for now.

I'm finding that Oracle's vertx client isn't setting column descriptors when there's an update with "remove" query. Bunch of these failures still there. 16 tests failures I think.
```
        Caused by:
        java.lang.IndexOutOfBoundsException: Index 0 out of bounds for length 0
            at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
            at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:70)
            at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:248)
            at java.base/java.util.Objects.checkIndex(Objects.java:372)
            at java.base/java.util.ArrayList.get(ArrayList.java:459)
            at org.hibernate.reactive.adaptor.impl.ResultSetAdaptor$1.getColumnType(ResultSetAdaptor.java:390)
            at org.hibernate.loader.custom.JdbcResultMetadata.getHibernateType(JdbcResultMetadata.java:68)
            at org.hibernate.loader.custom.ScalarResultColumnProcessor.performDiscovery(ScalarResultColumnProcessor.java:45)
            at org.hibernate.loader.custom.CustomLoader.autoDiscoverTypes(CustomLoader.java:494)
            at org.hibernate.reactive.loader.custom.impl.ReactiveCustomLoader.discoverTypes(ReactiveCustomLoader.java:123)
            at org.hibernate.reactive.loader.ReactiveLoader.lambda$doReactiveQueryAndInitializeNonLazyCollections$0(ReactiveLoader.java:76)
            at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1072)
```

Reply:
Ok, but why do you need it?

Reply:
Ah, wait, nevermind

Reply:
@tsegismon Does it make sense for `io.vertx.sqlclient.SqlResult#columnDescriptor()` to be empty?

`SqlResult#columnsNames()` return the expected result.
The query doesn't return any result (that's correct).

This is the query:
```
SELECT * FROM Person_phones where Person_id = ?
```

and  the table:
```
create table Person_phones (Person_id number(10,0) not null, "number" varchar2(255 char), "type" varchar2(255 char))
```

Reply:
I've created an issue for Vert.x SQL client: https://github.com/eclipse-vertx/vertx-sql-client/issues/1068

Reply:
I'll take a look asap, thank you @DavideD